epha/robot
# Feature swissmedic history

How to test this feature:

I think the easiest way to test this is to run `npm run robot-service` and check the swissmedic history file. However there aren't quite rarely changes.
I would recommend to merge this pull request as it shouldn't have any effect on pre-pull request functionality.
